src/devel.rs

```diff
@@ -122,7 +123,11 @@ pub async fn gendb(config: &mut Config) -> Result<()> {
 
         !devel_info.info.contains_key(pkg)
     });
-    println!("{} {}", action.paint("::"), bold.paint("Querying AUR..."));
+    println!(
+        "{} {}",
+        action.paint("::"),
+        bold.paint(tr!("Querying AUR...)"))
```

Comment:
What's this closing bracket? Is this expected?
```suggestion
        bold.paint(tr!("Querying AUR..."))
```

Reply:
Probably just an error from refactoring.